extract/extract.py

```diff
@@ -568,51 +602,204 @@ def decode_vbe(data):
+                91: "\x64\x23\x26",
+                92: "\x4D\x5A\x76",
+                93: "\x52\x5B\x25",
+                94: "\x63\x6C\x24",
+                95: "\x3F\x48\x2B",
+                96: "\x7B\x55\x28",
+                97: "\x78\x70\x23",
+                98: "\x29\x69\x41",
+                99: "\x28\x2E\x34",
+                100: "\x73\x4C\x09",
+                101: "\x59\x21\x2A",
+                102: "\x33\x24\x44",
+                103: "\x7F\x4E\x3F",
+                104: "\x6D\x50\x77",
+                105: "\x55\x09\x3B",
+                106: "\x53\x56\x55",
+                107: "\x7C\x73\x69",
+                108: "\x3A\x35\x61",
+                109: "\x5F\x61\x63",
+                110: "\x65\x4B\x50",
+                111: "\x46\x58\x67",
+                112: "\x58\x3B\x51",
+                113: "\x31\x57\x49",
+                114: "\x69\x22\x4F",
+                115: "\x6C\x6D\x46",
+                116: "\x5A\x4D\x68",
+                117: "\x48\x25\x7C",
+                118: "\x27\x28\x36",
+                119: "\x5C\x46\x70",
+                120: "\x3D\x4A\x6E",
+                121: "\x24\x32\x7A",
+                122: "\x79\x41\x2F",
+                123: "\x37\x3D\x5F",
+                124: "\x60\x5F\x4B",
+                125: "\x51\x4F\x5A",
+                126: "\x20\x42\x2C",
+                127: "\x36\x65\x57",
+            }
+
+            d_combination = {
+                0: 0,
+                1: 1,
+                2: 2,
+                3: 0,
+                4: 1,
+                5: 2,
+                6: 1,
+                7: 2,
+                8: 2,
+                9: 1,
+                10: 2,
+                11: 1,
+                12: 0,
+                13: 2,
+                14: 1,
+                15: 2,
+                16: 0,
+                17: 2,
+                18: 1,
+                19: 2,
+                20: 0,
+                21: 0,
+                22: 1,
+                23: 2,
+                24: 2,
+                25: 1,
+                26: 0,
+                27: 2,
+                28: 1,
+                29: 2,
+                30: 2,
+                31: 1,
+                32: 0,
+                33: 0,
+                34: 2,
+                35: 1,
+                36: 2,
+                37: 1,
+                38: 2,
+                39: 0,
+                40: 2,
+                41: 0,
+                42: 0,
+                43: 1,
+                44: 2,
+                45: 0,
+                46: 2,
+                47: 1,
+                48: 0,
+                49: 2,
+                50: 1,
+                51: 2,
+                52: 0,
+                53: 0,
+                54: 1,
+                55: 2,
+                56: 2,
+                57: 0,
+                58: 0,
+                59: 1,
+                60: 2,
+                61: 0,
+                62: 2,
+                63: 1,
+            }
```

Comment:
This is arguably still reads poorly because of the scrolling it causes. Maybe rows of 8 or 10, keeping the separate line for { and }, would be more readable than the original while still compact enough to fit on a screen

Reply:
Yes, I'll check how to force the formatter to ignore those sections (d_decode and d_combination). I agree they were better like they were before.

Reply:
That should be nicer to look at!